tamr_unify_client/models/project/categorization.py

```diff
@@ -15,4 +16,28 @@ def model(self):
         alias = self.api_path + "/categorizations/model"
         return MachineLearningModel(self.client, None, alias)
 
+    def create_taxonomy(self, name=""):
```

Comment:
This should take a `creation_spec` even though it just contains a `name` right now.
As the API adds features to taxonomies, we don't want existing code to break.